src/app/workspace/organization/users/organization-users.component.ts

```diff
@@ -17,10 +17,57 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import {Component} from '@angular/core';
+import {Component, OnInit} from '@angular/core';
+import {Store} from '@ngrx/store';
+import {AppState} from '../../../core/store/app.state';
+import {Observable} from 'rxjs/Observable';
+import {selectOrganizationFromUrl} from '../../../core/store/organizations/organizations.state';
+import {OrganizationModel} from '../../../core/store/organizations/organization.model';
+import {UserModel} from '../../../core/store/users/user.model';
+import {selectAllUsers} from '../../../core/store/users/users.state';
+import {tap} from 'rxjs/operators';
+import {selectAllGroups} from '../../../core/store/groups/groups.state';
+import {GroupModel} from '../../../core/store/groups/group.model';
+import {UsersAction} from '../../../core/store/users/users.action';
+import {OrganizationsAction} from '../../../core/store/organizations/organizations.action';
 
 @Component({
-  templateUrl: './organization-users.component.html'
+  templateUrl: './organization-users.component.html',
+  styleUrls: ['./organization-users.component.scss']
 })
-export class OrganizationUsersComponent {
+export class OrganizationUsersComponent implements OnInit {
+
+  public organization$: Observable<OrganizationModel>;
+  public users$: Observable<UserModel[]>;
+  public groups$: Observable<GroupModel[]>;
+
+  constructor(private store: Store<AppState>) {
+  }
+
+  public ngOnInit() {
+    this.organization$ = this.store.select(selectOrganizationFromUrl);
+    this.users$ = this.store.select(selectAllUsers).pipe(tap(this.sortUsers));
```

Comment:
Why do you use `tap` operator here? It should be used only for some side effects. It is better to use `map` here so you can directly work with the result and do not rely on the sort operation being done "in the background".